change/@ni-nimble-components-efb8d221-fd75-4cab-ae9d-fb2e7458ffec.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Technically this is a breaking change to our table page object's API, but it's incubating and there are no clients using it.